wcag20/sources/techniques/css/C7.xml

```diff
@@ -30,8 +30,9 @@ a span {
 height: 1px;
 width: 1px;
 position: absolute;
+clip: rect(0 0 0 0);
+margin: -1px;
```

Comment:
That can be solved with background: none|transparent; right?

—Michiel

> On 11 Jan 2016, at 01:00, Andrew Kirkpatrick <notifications@github.com> wrote:
> 
> In wcag20/sources/techniques/css/C7.xml:
> 
> > @@ -30,8 +30,9 @@ a span {
> >  height: 1px;
> >  width: 1px;
> >  position: absolute;
> > +clip: rect(0 0 0 0);
> > +margin: -1px;
> This thread discusses it: h5bp/html5-boilerplate#194 (in short, to avoid an errant single pixel of an item with a background color)
> 
> —
> Reply to this email directly or view it on GitHub.
>